examples/multiphysics/rayleigh_taylor_instability/rayleigh_taylor_instability.prm

```diff
@@ -0,0 +1,165 @@
+  set grid arguments 		= 1, 4 : 0.25, 1 : 0 , 0 : true
+  set initial refinement 	= 7
+end
+
+#---------------------------------------------------
+# Mesh Adaptation
+#---------------------------------------------------
+subsection mesh adaptation
+  set type                    	= kelly
+  set variable                	= phase
+  set fraction type           	= fraction
+  set max refinement level    	= 7
+  set min refinement level    	= 5
+  set frequency               	= 1
+  set fraction refinement     	= 0.9
+  set fraction coarsening     	= 0.001
+end
+
+#---------------------------------------------------
+# Multiphysics
+#---------------------------------------------------
+subsection multiphysics
+  set fluid dynamics		= true
+  set VOF 			= true
+end
+
+#---------------------------------------------------
+# VOF
+#---------------------------------------------------
+subsection VOF
+  subsection interface sharpening
+    set enable      		= true
+    set threshold		= 0.5
+    set interface sharpness	= 1.5
+    set frequency		= 25
+    set type        		= constant
+  end
+ 
+  subsection mass conservation
+    set monitoring          	= true
+    set monitored fluid     	= fluid 1
+    set tolerance           	= 1e-6
+    set verbosity           	= quiet
+  end
+
+end
+
+# --------------------------------------------------
+# Boundary Conditions
+#---------------------------------------------------
+subsection boundary conditions
+  set number                  = 4
+    subsection bc 0
+    set id = 0
+        set type              = periodic
+        set periodic_id	      = 1
+        set periodic_direction = 0
+    end
+    subsection bc 1
+    set id = 1
+        set type              = periodic
+        set periodic_id	      = 0
+        set periodic_direction = 0
+    end
+    subsection bc 2
+    set id = 2
+        set type              = noslip
+    end
+    subsection bc 3
+    set id = 3
+        set type              = noslip
+    end
+end
+
+# --------------------------------------------------
+# Source term
+#---------------------------------------------------
+subsection source term
+  set enable                 = true
+    subsection xyz
+            set Function expression = 0 ; -9.81 ; 0
+    end
+end
+
+# --------------------------------------------------
+# Non-Linear Solver Control
+#---------------------------------------------------
+subsection non-linear solver
+  set verbosity               = verbose
+  set tolerance               = 1e-8
+  set max iterations          = 50
+end
+
+# --------------------------------------------------
+# Linear Solver Control
+#---------------------------------------------------
+subsection linear solver
+  set verbosity					= verbose
+  set method                  			= gmres
+  set max iters               			= 10000
+  set relative residual                      	= 1e-8
+  set minimum residual                       	= 1e-10
+  set ilu preconditioner fill 			= 0
```

Comment:
remove unnecessary parameters (from `ilu preconditioner fill` to the end of this subsection)

Reply:
Bruno modified this subsection.